config/environments/cucumber.rb

```diff
@@ -45,4 +45,7 @@
     ENV_JAVA['https.proxyHost'] = nil
     ENV_JAVA['https.proxyPort'] = nil
   end
+
+  # load WIP features flag
+  config.deploy_wip_pipelines = true
```

Comment:
Do we want this?

Reply:
It is enabled for test and development environments. I would like to align it with the cucumber environment as well.

Reply:
Doesn't that mean that the Cucumber tests run when creating a production build also uses WIP pipelines?

Reply:
Firefox is usually a go to browser for testing but I have not looked into that. It can be started as an alternative browser option. Every chrome and driver version we keep changing the options to initialise them to use through Selenium. It is supposed to be unifying access to different browsers.